js/activityDetection/index.js

```diff
@@ -5,10 +5,13 @@ const throttle = require('../utils/throttle');
 const log = require('../log.js');
 const logLevels = require('../enums/logLevels.js');
 
-let setIsAutoReload;
+let setIsAutoReloadFnc;
+let getIsOnlineFnc;
 if (!process.env.ELECTRON_QA) {
     /* eslint-disable global-require */
-    setIsAutoReload = require('../windowMgr').setIsAutoReload;
+    const { getIsOnline, setIsAutoReload } = require('../windowMgr');
+    getIsOnlineFnc = getIsOnline;
+    setIsAutoReloadFnc = setIsAutoReload;
     /* eslint-enable global-require */
```

Comment:
is this relevant anymore?

Reply:
Yes, as `windowMrg` is not `global-required` to prevent the unit test from failing